Stripe/STPBankAccount.m

```diff
@@ -70,8 +86,16 @@ + (instancetype)decodedObjectFromAPIResponse:(NSDictionary *)response {
     bankAccount.country = dict[@"country"];
     bankAccount.fingerprint = dict[@"fingerprint"];
     bankAccount.currency = dict[@"currency"];
-    bankAccount.validated = [dict[@"validated"] boolValue];
-    bankAccount.disabled = [dict[@"disabled"] boolValue];
+    NSString *status = dict[@"status"];
+    if ([status isEqual: @"new"]) {
+        bankAccount.status = STPBankAccountStatusNew;
+    } else if ([status isEqual: @"validated"]) {
```

Comment:
maybe not a road we need to cross now, but it would be nice to have a generic enum serializer/deserializer protocol thingy